Taskfile.yml

```diff
@@ -65,13 +65,13 @@ tasks:
     - vagrant ssh-config > ssh.config.vagrant
     # For some reason, vagrant adds "LogLevel FATAL" to the generated file.
     # We want the default logging level instead.
-    - sed -i "" 's/LogLevel FATAL//' ssh.config.vagrant
```

Comment:
this is the only thing I am not sure... I think this is due to: mac vs linux... You were using sed on darwin which can behave differently than sed on ubuntu...

Reply:
In my understanding this will work also for Linux, while if you change it, it will break macOS.
What happens on Linux if you leave it as-is ?

Reply:
If it was working on Linux I would not touch the line :)

> sed: can't read s/LogLevel FATAL//: No such file or director

Reply:
Sigh. Could you please try this then
```
- sed -i bak 's/LogLevel FATAL//' ssh.config.vagrant
- rm ssh.config.vagrant.bak
```

Reply:
This works:
```
- sed --in-place=".bak" 's/LogLevel FATAL//' ssh.config.vagrant
- rm ssh.config.vagrant.bak
```

I couldn't make it work with: `-i bak` even though the man page says that `-i` and and `--in-place` are the same.
> sed: can't find label for jump to `ak'

It works with `-i=.bak`, but the bkp filename is `ssh.config.vagrant=.bak`...

Reply:
I updated the PR.. can you test on mac?